frontend/javascripts/viewer/model/sagas/mesh_saga.ts

```diff
@@ -527,6 +528,8 @@ function* maybeLoadMeshChunk(
         segmentMeshController.removeMeshById(segmentId, layer.name);
       }
 
+      const opacity = meshExtraInfo.opacity || Constants.DEFAULT_MESH_OPACITY;
```

Comment:
```suggestion
      const opacity = meshExtraInfo.opacity ?? Constants.DEFAULT_MESH_OPACITY;
```

otherwise, an opacity of 0 will not take effect. I'm wondering whether we can avoid the usage of DEFAULT_MESH_OPACITY here, though. I've seen the default variable also in the action creator and in the reducer. ideally, there would only be one place where the default value is used. you could achieve this by allowing "undefined" as a valid opacity parameter and then defaulting in the reducer (or action creator).